fpdcc/ccfp-asset-dashboard
# Flatten CIP tables based on phase funding streams + add more data to CIP export

## Overview

- related to #257, but doesn't close it. This first step of flattening the data is needed in order to add the filters. 
- closes #219

## Testing Instructions
- visit the cip planner: https://ccfp-asset-d-feature-fl-3knsmw.herokuapp.com/
  - login with username=`test` / password=`supersecret`
- build a portfolio. see how the columns are different and the phase funding stream make each row unique
- export the portfolio and make sure the data looks right
